trojsten/results/templates/trojsten/results/parts/results_table.html

```diff
@@ -74,7 +74,10 @@
             {{ row.user.school.name }}
```

Comment:
Myslím, že toto `School.name` momentálne vôbec nie je definované a čudujem sa, ako toto vôbec funguje: https://github.com/trojsten/web/blob/master/trojsten/schools/models.py
Skús otestovať, čo sa teda stane ak škola nemá skratku, ak to teda padne
 asi by to bolo vhodné nahradiť tým istým ako v zozanme účastníkov sústerdka.

Reply:
Myslim ze je to odtialto:
https://github.com/trojsten/web/blob/master/trojsten/results/representation.py#L80